wordle-web/src/components/ScoreDialog.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <v-dialog :model-value="modelValue" @update:model-value="close" :max-width="300">
+    <v-card>
+      <div class="d-flex justify-center pa-2 bg-primary text-h6">
+        You {{ gameResult?.success ? 'Win' : 'Loose' }}
+      </div>
+      <v-card-text class="d-flex flex-column align-center">
+        <v-row dense>
+          <v-col cols="6" class="text-end">Score -</v-col>
+          <v-col cols="6">{{ gameResult?.score }}</v-col>
+          <v-col cols="6" class="text-end">Attempts -</v-col>
+          <v-col cols="6">{{ gameResult?.attempts }}</v-col>
+          <v-col cols="6" class="text-end">Duration -</v-col>
+          <v-col cols="6">{{ duration }}</v-col>
+        </v-row>
```

Comment:
Typo in 'Lose' at the top.
Also, you might be able to use a v-for instead of manually creating each row. I was going to test out an idea but the score dialog is currently broken.